sktime/forecasting/base/_base.py

```diff
@@ -182,7 +182,14 @@ def fit(self, y, X=None, fh=None):
 
         return self
 
-    def predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+    def predict(
+        self,
+        fh=None,
+        X=None,
+        return_pred_int=False,
+        alpha=DEFAULT_ALPHA,
+        old_predict_interval_logic=True,
```

Comment:
I think this should be a tag, not an argument - it indicates, for any concrete forecaster, whether the logic for prediction intervals from `_predict` has been moved to `_predict_interval`.